var/spack/repos/builtin/packages/apple-gl/package.py

```diff
@@ -38,7 +38,5 @@ class AppleGl(AppleGlBase):
 
     provides("gl@4.1")
 
-    requires(
-        "platform=darwin %apple-clang",
-        msg="Apple-GL is only available on Darwin, when using Apple Clang",
-    )
+    depends_on("apple-clang", type="build")
+    requires("platform=darwin", msg="Apple-GL is only available on Darwin, when using Apple Clang")
```

Comment:
```suggestion
    requires("platform=darwin", msg="Apple-GL is only available on Darwin")
```

We should be able to drop the build dependency on `apple-clang` since this is a stub package that comes prebuilt from Apple. It likely required `%apple-clang` previously as a way of requiring darwin and not intermixing runtimes if someone built a GCC. I don't think that'll be an issue now so we should be good to delete this.